airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -832,6 +835,28 @@
         }
     }
 
+    function showMediaSessionMetadata(song) {
+        if ('mediaSession' in top.playQueue.navigator) {
+            var metadata = new MediaMetadata({
+                title: song.title,
+                artist: song.artist,
+                album: song.album,
+                artwork: [
+                    // The "&size=" parameter always resizes the image to a JPEG image (even if the original is PNG).
+                    { src: song.coverArtUrl + "&size=96", sizes: '96x96', type: 'image/jpeg' },
```

Comment:
Don't remember off the top of my head, but make sure there is a covertArtUrl property of song?

Reply:
There is actually, so it's working as intended. My OS doesn't display images in notifications though, so I can't "really" test it ;)